i18n/zh/docusaurus-plugin-content-docs/current/design-document/03-connect/11-kafka-connector.md

```diff
@@ -47,7 +47,7 @@ pubSubConfig:
   passWord: kafkaPassWord
 connectorConfig:
   connectorName: kafkaSource
-  # kafka连接参数 ↓
+  # kafka 连接参数 ↓
   bootstrapServers: 127.0.0.1:9092
```

Comment:
I noticed that in https://github.com/apache/eventmesh-site/compare/4432a124c26ed09e036c1ff837744e46d8944def..82acf2ed071c0c098faf9b1feff2613c79e1ba44, the content within the code block has been reformatted again. Why has this occurred?

Reply:
Oh, I am installed wrong version of AutoCorrect, it not have that feature.